src/nifty-gateway.ts

```diff
@@ -1,5 +1,5 @@
 import { BuilderInstanceCreated } from "../generated/NiftyGateway/NiftyGateway"
-import { NiftyNFT } from "../generated/NiftyGateway/NiftyNFT"
```

Comment:
That's interesting, as I don't have any problems locally. And actually, if I try to change the path to `/generated/templates/NftContract/NiftyNFT` it gives me an error. Could you please show me the content of your `generated/templates/NftContract` folder?

Reply:
I figured out the problem, it failed indeed on older node versions:
https://github.com/frostiq/cryptomedia-subgraph/actions/runs/476947450

Kudos to @developerfred for adding a build pipeline with multiple node versions!